src/migrations/20201108154514-create-rooms.js

```diff
@@ -19,6 +19,10 @@ module.exports = {
       checkOut: {
         type: Sequelize.DATE,
       },
+      Status: {
+        type: Sequelize.STRING,
+        defaultValue: "inactive",
```

Comment:
it should be active by default...if a user is checked out, it will be updated to inactive